declarations/index.d.ts

```diff
@@ -10,6 +10,7 @@ export { atr, ATR } from './directionalmovement/ATR';
 export { truerange, TrueRange } from './directionalmovement/TrueRange';
 export { roc, ROC } from './momentum/ROC';
 export { kst, KST } from './momentum/KST';
+export { psar, PSAR } from './momentum/PSAR';
```

Comment:
Not sure about the directory location?

Reply:
for now let it be there we can move it where appropriate